0000-catalog.md

```diff
@@ -0,0 +1,789 @@
+- Feature Name: Product Catalog
+- Start Date: 2019-07-12
+- RFC PR:
+- Hyperledger Grid Issue: [GRID-95](https://jira.hyperledger.org/browse/GRID-95)
+
+# Summary
+
+[summary]: #summary
+
+This RFC proposes a generic implementation for a Hyperledger Grid _Product
+Catalog_. Product Catalogs will be represented by a unique identifier, which
+will be referenced by the _catalog_products_ included in the catalog. That
+catalog_id can be used to share a grouping of products that can be shared with 
+one or more organizations. The Grid Catalog will contain 4 fields. A 
+**catalog_id**, a **catalog_owner**, a **Name** for the catalog, and a repeated 
```

Comment:
I noticed some inconsistencies with the capitalization and text emphasis of titles, messages, and field names throughout the RFC. 

For example, I think **Name** be lowercase here. Also, in the next paragraph down they have an italics emphasis applied. I'd probably recommend using the `code` emphasis instead for these to make it clear that you're referring to actual fields in the messages.

I think a quick pass through to clean those up would make the RFC a lot more clear!

Reply:
✅  updated, let me know if I missed anything